lib/rubocop/cop/style/empty_literal.rb

```diff
@@ -60,9 +78,11 @@ def on_send(node)
 
         def offense_message(node)
           if offense_array_node?(node)
-            ARR_MSG
+            current = array_with_index(node) ? 'Array[]' : 'Array.new'
+            format(ARR_MSG, current: current)
```

Comment:
Can you update as below and adjust the specs to have more appropriate message?

```suggestion
            format(ARR_MSG, current: node.source)
```